modules/apps/portal-language/portal-language-override-service/src/main/java/com/liferay/portal/language/override/service/impl/PLOEntryLocalServiceImpl.java

```diff
@@ -222,8 +229,14 @@ private void _validate(String key, String languageId, String value)
 			throw new PLOEntryKeyException.MustBeShorter(keyMaxLength);
 		}
 
-		if (!ArrayUtil.contains(PropsValues.LOCALES, languageId)) {
-			throw new PLOEntryLanguageIdException.MustBeAvailable(languageId);
+		Set<Locale> availableLocales = _language.getAvailableLocales();
+
+		String[] availableLanguageIds = TransformUtil.unsafeTransform(
+			availableLocales.toArray(), Object::toString, String.class);
```

Comment:
If you search other usages, you'll usually find one of these two types of method references for the transform:
- `Locale::toString` (explicit type)
- `String::valueOf`
Either is fine with me, but `Object::toString` feels somehow too broad and too specific at the same time.

Reply:
Ops, sorry. I searched for `Locale::toString`, but haven't found any usage of that. So I believe you, generically speaking, meant `ClassName::MethodName`.

Either way, you are right. But in this case, Locale::toString gives me an 
error message.
![image](https://github.com/user-attachments/assets/a5e2e108-09e0-4e6a-89c4-fafa2a059736)

Which is weird to me, since `Locale::toString` overrides `Object::String`.

![image](https://github.com/user-attachments/assets/12b4919e-b16f-4e65-8cdf-bbe51f5c3830)

Reply:
Ah, interesting, so there's an ambiguous overload. In that case, `String::valueOf` is the way to go I think.